syngenta_digital_dta/s3/adapter.py

```diff
@@ -55,7 +55,8 @@ def put(self, **kwargs):
             ACL=acl,
             Body=body,
             Bucket=self.bucket,
-            Key=kwargs['s3_path']
+            Key=kwargs['s3_path'],
+            Tagging=kwargs.get('tags', 'name=default')
```

Comment:
don't force that tag name=default in case the caller doesn't pass the tags, Tagging property in this case should be empty string or None if the utility accepting that or you shouldn't pass the property at all